### PR TITLE
docs: update packages that can be removed

### PR DESCRIPTION
**Pull request by brandonocasey**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -139,6 +140,17 @@ semver
...
+lint-staged
+not-prerelease
+npm-merge-driver-install
+npm-run-all
+shx
+pkg-ok
+in-publish
 ```
 
 All these dependencies should be removable because, in general, build tools are now using the new `videojs-generate-*` libraries. This should make it easier to manage dependencies by offloading much of that work to the Video.js organization. The behavior of these new tools is customizable by editing the files in `scripts/`.
```
Comment (gkatsev) [inline README.md]: can we list out the specific videojs-generate packages that replace the above here?

Comment (brandonocasey) [reply README.md]: sure

File: README.md [hunk 2]
```diff
@@ -111,45 +119,87 @@ If you have made no customizations, the simplest process may be to remove some o
...
+rollup-plugin-commonjs
+rollup-plugin-json
+rollup-plugin-multi-entry
+rollup-plugin-node-resolve
+rollup-plugin-uglify
+rollup-plugin-terser
+uglify-es
+```
+
+#### `videojs-genreate-karma-config`
```
Comment (misteroneill) [inline README.md]: Typo: `genreate`

Comment (brandonocasey) [reply README.md]: fixed

**Pull request merged**